docs/quick-ingestion-guides/Page_3-Configuration.md

```diff
@@ -0,0 +1,114 @@
+# DataHub [BigQuery] UI Ingestion Guide: Configuring Your [BigQuery] to DataHub Connector
+
+## UI Ingestion
+
+* Navigate to the "Ingestion Tab"
+![Navigate to the "Ingestion Tab"](../imgs/quick-ingestion-guide/bigquery_ingestion_ingestion_button.png)
+
+## Step 1: Configure Secrets
+1. Select `Secrets tab`
+![Secrets Tab](../imgs/quick-ingestion-guide/bigquery_ingestion_secrets_tab.png)
```

Comment:
Couple of things here & all images -- 

1. Follow formatting/sizing best practices here: https://github.com/datahub-project/datahub/blob/master/docs/README.md#docs-best-practices
2. Reference images in `datahub-project/static-assets` repo